ui/src/app/components/runtime-configuration-panel.spec.tsx

```diff
@@ -867,10 +884,14 @@ describe('RuntimeConfigurationPanel', () => {
   it('should warn user about reboot if there are updates that require one - increase disk size', async () => {
     const wrapper = await component();
 
-    await pickMainDiskSize(wrapper, getMainDiskSize(wrapper) + 10);
+    await pickDetachableDiskSize(wrapper, getDetachableDiskSize(wrapper) + 10);
     await mustClickButton(wrapper, 'Next');
 
-    expect(wrapper.find(WarningMessage).text().includes('reboot')).toBeTruthy();
+    // After https://precisionmedicineinitiative.atlassian.net/browse/RW-9167 Re-attachable persistent disk is default
```

Comment:
this test is no longer valid.  Please do this:
1. update the text to accurately reflect what this test does
2. implement a new test that does what this one claims to do: `should warn user about reboot if there are updates that require one - increase disk size`

Reply:
Done